app/views/users/edit.html.erb

```diff
@@ -1,15 +1,20 @@
+<%if @user.image%>
+      <%= image_tag @user.image, alt: "Profile Pic", size: "150x200"%>
+    <%else%>
+      <%= image_tag "http://www.eurogeosurveys.org/wp-content/uploads/2014/02/default_profile_pic.jpg", alt: "Profile Pic", size: "150x200"%>
```

Comment:
Can we store this default image in the user as the default image if one is not set? Hardcoding it here feels weird, and the conditional logic is pretty complicated given the only thing that changes is the `src`.